README.md

```diff
@@ -204,13 +204,13 @@ See instructions in their [documentation](https://docs.wavesplatform.com/en/wave
 
 ## 2. Node extension installation
 
-Since a version **2.3.0** Matcher has been using `grpc-server` extension from the Node to get data with a blockchain events and updates.
+Since a version **2.3.0** Matcher has been using `waves-grpc-server` extension from the Node to get data with a blockchain events and updates.
```

Comment:
Since a version **2.3.4** Matcher has been using `waves-grpc-server` extension from the Node to get data with a blockchain events and updates.
Since a version **2.3.0** until **2.3.4** Matcher used `grpc-server` extension (the old name for `waves-grpc-server`).